docs/release-notes/8.8.asciidoc

```diff
@@ -58,7 +58,6 @@ To view a detailed summary of the latest features and enhancements, check out ou
 [[breaking-changes-8.8.0]]
 ==== Breaking changes
 
-//tag::breaking-changes[]
 // NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
 // THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
 :pull: https://github.com/elastic/kibana/pull/
```

Comment:
I'd remove these lines too. Since this is no longer a snippet/tagged region, we don't need reminders in this particular region on how a snippet works (we also have them in the actual snippet above).

We also don't need to redefine the `pull` attribute here since this is no longer a snippet. But we might want to move the `pull` line to the snippet above -- it's not technically necessary because there aren't any links in that snippet, but it's probably a good idea to keep it with the rest of the snippet structure so it'll be easier for future use.

Reply:
Ok, can do. No objections to removing the comments -- I was wondering if we should keep them. Agree with moving the pull attribute up to 8.8.1. Even if there's no use for it at the moment, it's just good to keep it grouped for future use.